extensions/amp-story/1.0/amp-story.css

```diff
@@ -197,7 +197,7 @@ amp-story:not([desktop]) >
 }
 
 amp-story:not([desktop]) >
-    amp-story-page.i-amphtml-layout-container[distance="1"] {
+    amp-story-page:not[active].i-amphtml-layout-container[distance="1"] {
```

Comment:
When can this happen?  Is this when you navigate away from a page, it can lose the `active` attribute before the distance gets changed?